docs: fixing typos in multiple files (#6653)
projectdiscovery/nuclei: pkg/tmplexec/flow/README.md

**1. Protocol Execution Functions**

>   Any protocol that is present in a nuclei template can be called/executed in javascript in format `proto_name()` i.e `http()` , `dns()` , `ssl()` etc.

Reviewer: ```suggestion
  Any protocol that is present in a nuclei template can be called/executed in JavaScript using the format `proto_name()`, e.g., `http()` , `dns()` , or `ssl()`.
```
